### PR TITLE
Arafat branch

### PR DESCRIPTION
**Pull request by Fatmangh**
I have fixed the status issue. Now status is saved into datastore and loaded whenever necessary. Also taken care of some style issues.

### CODE REVIEW

File: src/main/java/codeu/controller/ConversationServlet.java [hunk 1]
```diff
@@ -40,9 +40,9 @@
   private ConversationStore conversationStore;
 
   /**
-   * Set up state for handling conversation-related requests. This method is only called when
```
Comment (tbiscuit) [inline src/main/java/codeu/controller/ConversationServlet.java:43]: It doesn't look like this file needs to change, best to just revert it.

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 2]
```diff
@@ -37,9 +37,9 @@
   private ConversationStore conversationStore;
 
   /**
-   * Set up state for handling conversation-related requests. This method is only called when
-   * running in a server, not when running in a test.
-   */
```
Comment (tbiscuit) [inline src/main/java/codeu/controller/ProfileServlet.java:42]: These indents didn't need to change.

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 3]
```diff
@@ -48,31 +48,30 @@ public void init() throws ServletException {
...
-   * This function fires when a user navigates to the conversations page. It gets all of the
-   * conversations from the model and forwards to profiles.jsp for rendering the list.
-   */
+  * This function fires when a user navigates to the profile page. It gets all of the
+  * conversations and all relevant info from the model and forwards to profiles.jsp for rendering the list.
+  */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
-      throws IOException, ServletException {
+  throws IOException, ServletException {
```
Comment (tbiscuit) [inline src/main/java/codeu/controller/ProfileServlet.java:72]: Google style is to indent this line (the way it was)

File: src/main/java/codeu/controller/Upload.java [hunk 4]
```diff
@@ -19,56 +19,56 @@
...
+  void setUserStore(UserStore userStore) {
+    this.userStore = userStore;
+  }
 
-    @Override
-    public void doPost(HttpServletRequest req, HttpServletResponse res)
-        throws ServletException, IOException {
+  @Override
+  public void doPost(HttpServletRequest req, HttpServletResponse res)
+  throws ServletException, IOException {
```
Comment (tbiscuit) [inline src/main/java/codeu/controller/Upload.java:46]: Indent line 46

**Review by tbiscuit — APPROVED** [review]
TL;DR I like the changes!

Usually one would have a style PR for all the indents you fixed and then a separate PR with the needed code fix. There's ways in Git to do this but since we're on the final week I won't ask for you to do the split, but FYI it is possible even after creating a PR to split it in case you find that there's really two things happening like style fixes + a needed code fix. Since my comments are all about the indents I'll approve, if you get time you can fix but overall the code looks better with this than it did before 👍
